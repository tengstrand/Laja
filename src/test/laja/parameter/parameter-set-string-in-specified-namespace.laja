
#write "target/test-output/parameter-set-string-in-specified-namespace.out"
  message = {message}
  $x.message = {$x.message}
#end
