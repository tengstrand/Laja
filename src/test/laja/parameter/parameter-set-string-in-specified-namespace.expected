  message = NULL
  $x.message = This is a string
